### ISSUE TITLE
Issues with text expander app (e.g: AutoHotKey, PhraseExpress)

### ISSUE THREAD

**Issue by Joroba3** [typora/typora-issues#1875]
I have found that Typora has issues with AHK's [hotstrings](https://autohotkey.com/docs/Tutorial.htm#s2). More specifically, the keyword will bu substituted, but the first letter will be left. For example, if _btw_ becomes _by the way_, typora will type _bby the way_.

**Comment by omattman** [comment]
Can confirm this is an ongoing issue. Will follow this thread.

**Comment by changhenry2** [comment]
Same issue, but in Source Code Mode this will not happen.

**Comment by DutchPete** [comment]
Can confirm this too.

**Comment by abnerlee** [comment]
Please reopen this if it still happens in latest version

**Comment by DutchPete** [comment]
On 1st Feb 2019 I switched to Linux Mint, and the 1st program not listed in the Software repository that I installed was ... Typora !! I am currently running version 0.9.65, which is the latest, if I am not mistaken.
AutoHotKey does not work in Linux, so I am using Autokey instead, which uses Python.

I can confirm that as of today this issue, as mentioned by Joroba3, is still very much there. To use his example:
*btw* will sometimes result in **b**by the way, sometimes in **btw**by the way.
